app/controllers/goodsRecord/SingleRecordController.scala

```diff
@@ -140,12 +139,24 @@ class SingleRecordController @Inject() (
             changedPage,
             pageRemoved,
             recordIsLocked,
-            para
+            para,
+            record.declarable,
+            getReviewReasonMessageKey(record.toReview, record.reviewReason)
           )
         ).removingFromSession(initialValueOfHasSuppUnit, initialValueOfSuppUnit)
       }
     }
 
+  private def getReviewReasonMessageKey(toReview: Boolean, reviewReason: Option[String]): Option[String] =
```

Comment:
I thought we were getting rid of this in place of reading the JSON string into relevant case models which we can use within the view? e.g https://github.com/hmrc/trader-goods-profiles-frontend/pull/612/files#diff-0a81a36ebf9cb747ed7bdcad81c16fa5efbdd3c38562a5179e273749bdb31d79R29